apps/prairielearn/src/models/questions.ts

```diff
@@ -0,0 +1,24 @@
+import sqldb = require('@prairielearn/postgres');
+import AnsiUp from 'ansi_up';
+
+const ansiUp = new AnsiUp();
+const sql = sqldb.loadSqlEquiv(__filename);
+
+export async function getQuestions(course_id, course_instances) {
```

Comment:
We typically use SQL verbs as a prefix, so `selectQuestions`; I'd go further and name this `selectQuestionsForCourse`.

You should add TypeScript types to the parameters of this function.

Reply:
done